src/components/Nav.jsx

```diff
@@ -24,6 +30,9 @@ export default function Nav() {
         <Link to="/topic/football">
       <button className="topic-button" onClick={handleClick} value="football">
         Football</button></Link>
+        <Link to='/sort_by/highest_rated'>
+        <button className="topic-button" onClick={handleQueryClick} value="highest_rated">
+        Highest Rated</button></Link>
```

Comment:
This should instead be some kind of dropdown to allow the user to select what they want to sort by. This should include date, votes, comment count and any other values that your api accepts. 

You should also have another drop down that allows the user to change the order by value, which should be either descending by default and optionally switchable to ascending.